### ISSUE TITLE
Theme installation failed.

### ISSUE THREAD

**Issue by miked137** [zionbuilder/zionbuilder#364]
Install of product failed

Unpacking the package…

Installing the theme…

The package could not be installed. The theme is missing the style.css stylesheet.

Theme installation failed.

**Comment by zauan** [comment]
Hi there,

Thank you for opening this ticket. This error is received because this is not a theme, it is a plugin that works with any theme.

Please install it as a plugin and let me know if it works.

Best regards,
Stefan

**Comment by miked137** [comment]
Zion builder Pro was not activated because it requires the Zion builder plugin.

**Comment by zauan** [comment]
Indeed, the Zion Builder free plugin can be installed from WordPress plugin directory ( https://wordpress.org/plugins/zionbuilder/ ). 

The PRO version adds additional functionality to the free version.

You can install the free plugin directly from WordPress dashboard > Plugins > Add new. On this page, you can search for the plugin and press the install button

Please let me know if you need further assistance

**Comment by miked137** [comment]
I just purchased the Pro version

I see 4 packages available in the downloads page from my account
zionbuilder-pro-v121
zionbuilder-pro-v120
zion-builder-pro_2.0.0_beta
zion-builder_2.0.0_beta

Which one do I use to install plugin and which one is the theme ?

**Comment by zauan** [comment]
You will need to install the Zion Builder FREE version from https://wordpress.org/plugins/zionbuilder/

Now, for the PRO plugin, please install the zionbuilder-pro-v121 as this is the latest version.

**Comment by miked137** [comment]
ok free version installed, now how to get Pro version installed

**Comment by zauan** [comment]
Please download the zionbuilder-pro-v121 zip file from the builder website and follow the following steps:

1. Log in to your WordPress dashboard and navigate to Plugins -> Add New
2. Press the Upload Plugin button at the top of the page
3. Click Choose File and select the plugin zip file downloaded earlier from your computer
4. Press Install Now
5. Once the installation is finished, press Activate Plugin

**Comment by miked137** [comment]
done, now what zip for the theme

**Comment by zauan** [comment]
The plugin can work with any theme. There is no theme provided with the plugin at the moment. This means that you can use any theme you want.

**Comment by miked137** [comment]
where do I add the demo's of all the websites then ?

**Comment by zauan** [comment]
I believe that there is confusion here. Can you please let me know what demos are you referring to?

**Comment by miked137** [comment]
KALLYAS WordPress Theme

**Comment by miked137** [comment]
Please cancel my service and refund my money, I assume what I purchased was just ZionBuilder Pro and was not through Kallyas which I thought it was

**Comment by miked137** [comment]
I just cancelled from your site, please refund purchase

**Comment by zauan** [comment]
Indeed, this is where the confusion arises. This is just a plugin that works with any theme, while Kallyas is a theme that comes with many demos and has its own builder. I will proceed with the refund.

Best regards,
Stefan

**Comment by miked137** [comment]
Thank you, I look forward to using those themes with your builder

Mike

**Comment by zauan** [comment]
I've finalized the refund. Please note that the Kallyas theme is using an older version of this plugin. The new major update for Kallyas that includes this updated builder is still in the works.

**Comment by miked137** [comment]
good to know thanks

**Comment by zauan** [comment]
You're welcome!

I wanted to mention this so that there won't be any confusion :)
